py3-click.yaml

```diff
@@ -41,7 +41,10 @@ subpackages:
     description: ${{vars.pypi-package}} installed for python${{range.key}}
     dependencies:
       provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
       runtime:
+        - python-${{range.key}}
```

Comment:
you dont have to do this.
melange adds the correct depend here.  (which is actually `python-${{range.key}}-base`)